utilities/boostHistHelpers.py

```diff
@@ -36,9 +36,7 @@ def divideHists(h1, h2, cutoff=1e-5, allowBroadcast=True, rel_unc=False, cutoff_
     # Apply cutoff to both numerator and denominator
     cutoff_criteria = np.abs(h2vals) > cutoff
     # By the argument that 0/0 = 1
-    out[(np.abs(h2vals) < cutoff) & (np.abs(h1vals) < cutoff)] = 1.
-    if not createNew:
-        out[cutoff_criteria] = cutoff_val
+    out[(np.abs(h2vals) < cutoff) & (np.abs(h1vals) < cutoff)] = cutoff_val
```

Comment:
Are you sure this is what you want? What is the use case?

The logic I had decided on was:

if numerator and denominator are both very small, set the ratio to 1.
if only denominator is small, set it to the cutoff value (because all bets are off on what the true value is)

What behaviour are you trying to get?

Reply:
Efficiencies, where denominator is small and numerator is consequently smaller (then aside from negative MC weights they should both be positive by construction, if not I set negative efficiencies to 0 myself where I compute them). So here the case 0/0 should correspond to efficiency 0 (since you would not have events anyway

Reply:
So is it ok or should I change something? The current behaviour for the ratio is just as it was before, the only difference is that you get to choose what to assign to 0/0 (default is still 1)